packages/testing/src/generators.ts

```diff
@@ -1,12 +1,23 @@
+import { randomBytes } from 'node:crypto';
+
 import type { Season } from '@packages/dates/config';
 import { getLastWeek, getCurrentWeek } from '@packages/dates/utils';
+import type { Address } from 'viem';
+import { keccak256 } from 'viem/utils';
 
 export const randomLargeInt = () => Math.floor(Math.random() * 1000000000) + 1000000000;
 
 export function randomIntFromInterval(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1) + min);
 }
 
+export function randomWalletAddress(): Address {
```

Comment:
Nice, keep needing random data

Reply:
this was orginally a method you added i think, i just moved it into generators.ts since this is where i've been putting random utils for testing :)